qemu/tests/balloon_check.py

```diff
@@ -103,8 +115,19 @@ def balloon_memory(self, new_mem):
         elif new_mem == 0:
             compare_mem = self.current_mmem
         elif new_mem <= 100:
-            self.current_mmem = self.get_ballooned_memory()
-            compare_mem = self.current_mmem
+            #got monitor ballooned memory more flexible for balloon_small_size test.
+            start_time = time.time()
+            mmem_rebound_timeout = 120
+            end_time = time.time() + float(mmem_rebound_timeout)
+            while time.time() < end_time:
```

Comment:
The two "while" in line 33 & 122 have very similar structure, better to merge it